data/deltarune/config.json

```diff
@@ -1,6 +1,12 @@
 {
     "game": "Deltarune",
-    "chapters": ["ch1", "ch2", "ch3", "ch4"],
+    "chapters": {
+        "init": "Chapter Select",
```

Comment:
GitHub Actions were not properly updated for this to work. It will fail due to a lack of `decompiled/init` directory.